common/network-common/src/main/java/org/apache/spark/network/server/TransportChannelHandler.java

```diff
@@ -176,4 +180,14 @@ public TransportResponseHandler getResponseHandler() {
     return responseHandler;
   }
 
+  @Override
+  public void channelRegistered(ChannelHandlerContext ctx) {
+    transportContext.getRegisteredConnections().inc();
```

Comment:
I think you need to call `super.blah` in these overrides so that other handlers in the pipeline also get the notification.